ext/soap/tests/bugs/bug27722.phpt

```diff
@@ -2,8 +2,7 @@
 Bug #27722 (Segfault on schema without targetNamespace)
 --EXTENSIONS--
 soap
---GET--
-wsdl
+--CGI--
```

Comment:
I don't quite understand why we need the CGI section in this text (and the other one)

Reply:
Oh, right, there's no need to run with php-cgi.